packages/modules/imodel-browser/src/containers/ITwinGrid/ITwinTile.tsx

```diff
@@ -86,6 +96,27 @@ export const ITwinTile = ({
             <ITwinIcon />
           </span>
         }
+        rightIcon={
+          isFavorite ? (
```

Comment:
couldnt this be condensed?
do the isFavorite check in the onClick callback and the Svg child?

```
<IconButton onClick(() => { isFavorite ? foo : bar })> {isFavorite ? Icon1 : Icon2 }</IconButton>
```